lte/gateway/python/integ_tests/s1aptests/test_attach_detach_maxbearers_twopdns.py

```diff
@@ -20,7 +20,7 @@
 from integ_tests.s1aptests.s1ap_utils import SpgwUtil
 
 
-class TestMaximumBearersTwoPdnsPerUe(unittest.TestCase):
+class TestAttachDetachMaxBearersTwoPdns(unittest.TestCase):
```

Comment:
Can we have definition name as test_attach_detach_maxbearers_two_pdns instead of test_attach_detach_maxbearers_twopdns

Reply:
The idea was to align the classname and function name with the filename here for clear debugging in CI with matching string names to search across. This suggested modification will need filename modification and updating of same in defs.mk file. This will be a bigger activity considering many other test cases with similar behavior and can be considered as a lower priority activity for future.